src/NServiceBus.Transport.SQS.AcceptanceTests/When_trying_to_subscribe_when_topic_does_not_exist.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NServiceBus.AcceptanceTesting.Customization;
+    using NServiceBus.AcceptanceTesting.Support;
+    using NServiceBus.Features;
+    using NUnit.Framework;
+
+    public class When_trying_to_subscribe_when_topic_does_not_exist : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_log_exception_with_topic_name()
+        {
+            var messageId = Guid.NewGuid();
+
+            await Scenario.Define<Context>(c =>
+            {
+                c.MessageId = messageId;
+                c.EnableInstallers = true;
+                c.DisableAutoSubscribe = true;
+            })
+            .WithEndpoint<Endpoint>()
+            .Done(context => true)
+            .Run();
+
+            var context = await Scenario.Define<Context>(c =>
+            {
+                c.MessageId = messageId;
+                c.EnableInstallers = false;
+                c.DisableAutoSubscribe = false;
+            })
+            .WithEndpoint<Endpoint>()
+            .Done(context => true)
+            .Run();
+
+            Assert.IsTrue(context.Logs.Any(a => a.Message.Contains("not found") && a.Message.Contains("When_trying_to_subscribe_when_topic_does_not_exist-MyEvent")));
```

Comment:
How about we assert the log level too?

Could we use a more expressive collection assert with a predicate? Something like

Assert.That(context.Logs, Has.One.Matches<ScenarioContext.LogItem>(x => x.Level ==  && ));

or whatever the NUnit thing is. Because that would make sure we get a more meaning message when the assertion fails.